decidim-comments/spec/models/comment_spec.rb

```diff
@@ -223,12 +223,6 @@ module Comments
           expect(parent.comment_threads.count).to eq 3
         end
 
-        it "returns 2 when a comment has been moderated" do
```

Comment:
Let's leave this here, please. I want to communicate that this is indeed the intention, and it isn't a bug. It should be "still returns 3 when a comment has been moderated". You'll need to also change the 2 for a 3 in line 229.